### PR TITLE
Add delete functionality to rest client

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: tests/ProviderDirectoryTest.php [hunk 1]
```diff
@@ -152,7 +152,7 @@ private function setupTestVarsAndTestGetListByHisp()
 
         foreach($needsToBeConfigured as $staticVar => $resultKey) {
             foreach($results as $result) {
-                if(!empty($result[$resultKey])) {
+                if(!empty($result[$resultKey]) && $result[$resultKey] != 'N/A') {
```
Comment (hendeavors) [inline tests/ProviderDirectoryTest.php:155]: Maxmd actually sends N/A back in the response?

Comment (ghost) [reply tests/ProviderDirectoryTest.php:155]: They didn't used to. But yeah, this broke a test I was checking for a non empty record to search for.

File: tests/RestClientTest.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?php
```
Comment (hendeavors) [inline tests/RestClientTest.php:1]: I don't know if mocking really works here. If maxmd doesn't work, neither does this library.

**Review by hendeavors — APPROVED** [review]

File: tests/RestClientTest.php (hunk 2 above)
Comment (ghost) [reply tests/RestClientTest.php:1]: It just tests that the abstract class works. It's testing all 3 get post and delete methods work in the base class as expected. The other pr in the proofing package tests making the delete call to maxmd.

**Pull request merged**